app/src/main/java/com/goldenraven/padawanwallet/ui/wallet/WalletScreen.kt

```diff
@@ -70,6 +74,9 @@ internal fun WalletScreen(
         )
     }
 
+    if (isOnline(LocalContext.current) && !Wallet.isBlockChainCreated())
+        Wallet.createBlockchain()
+
```

Comment:
This is a pretty big PR as is so I'll merge it like this, but let's add checks and banners for when network is not available as next steps.